plugins/polio/migrations/0198_remove_po_prefix.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.13 on 2024-10-04 13:18
+
+from django.db import migrations
+
+
+def remove_prefix_from_objects(objects):
+    for object in objects:
```

Comment:
Not sure how robust this is. it will only work for PO-XXX nothing else .. Not sure what we have in DB

Reply:
That's the only thing we have